python/ray/serve/_private/controller.py

```diff
@@ -974,6 +1014,14 @@ def _save_cpu_profile_data(self) -> str:
                 "the RAY_SERVE_ENABLE_CPU_PROFILING env var."
             )
 
+    def _get_logger_state(self) -> Dict:
+        """Test purpose, get the log information."""
```

Comment:
would like to avoid adding private methods for testing if possible, but if there's no other way it's OK
```suggestion
    def _get_logging_config(self) -> Dict:
        """Get the logging configuration (for testing purposes)."""
```
Would make this return a tuple instead of a dict with magic keys also